Classes/Core/Functional/Framework/Frontend/RequestBootstrap.php

```diff
@@ -104,6 +104,9 @@ private function setGlobalVariables()
 
         // Populating $_POST
         $_POST = [];
+        if ($this->request->hasHeader('Content-Type') && in_array('application/x-www-form-urlencoded', $this->request->getHeader('Content-Type'))) {
```

Comment:
What about `multipart/form-data`?

Reply:
Should also be added. I've currently added what I needed in my concrete use case.
Would need to check how that needs to be handled / parsed.

Should we add it within this PR, or would it be ok to not add it yet, but once someone needs that feature?

Reply:
I'd vote for adding this separately since I'm also not 100% sure what further implications this has.